PermissionsUtilsUI/src/main/java/com/drivequant/drivekit/permissionsutils/PermissionUtilsUI.kt

```diff
@@ -14,11 +22,17 @@ import com.drivequant.drivekit.permissionsutils.permissions.model.PermissionView
 object PermissionUtilsUI : PermissionsUtilsUIEntryPoint {
 
     internal var permissionViewListener: PermissionViewListener? = null
+    internal var isBluetoothNeeded:Boolean = true
+    internal var shouldDisplayDiagnosisLogs:Boolean = true
+    internal var contactType:ContactType = ContactType.NONE
+    internal var logPathFile: String = "/drivekit-permissions-utils/logs/"
 
+    @JvmStatic
```

Comment:
@mohamed-garci Why `@JvmStatic` annotation is used in this class ? Vehicle UI nor DriverData UI use it

Reply:
@Steven-DriveQuant You don't need to use INSTANCE if symbol is annotated as `@JvmStatic` when way you interact with it from java.

It can be applied for the other modules if this is okey. @jeremybayle-pro

Reply:
@mohamed-garci I agree with that but each `@JvmStatic` annotation on a method will increase number of generated methods (https://medium.com/@takusemba/do-you-need-jvmstatic-annotation-e21269277453). Not sure that's worth enough

Reply:
Keep singleton functions and remove @JvmStatic

Reply:
Done